web-animations/interfaces/Animatable/getAnimations.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Animatable.getAnimations tests</title>
+<link rel="help" href="https://w3c.github.io/web-animations/#dom-document-getanimations">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../../testcommon.js"></script>
+<body>
+<script>
+'use strict';
+
+test(function(t) {
+  var div = createDiv(t);
+  assert_array_equals(div.getAnimations(), []);
+}, 'Test getAnimations on element with no animations');
+
+test(function(t) {
+  var div = createDiv(t);
+  var animationA = div.animate(null, 100 * MS_PER_SEC);
+  var animationB = div.animate(null, 100 * MS_PER_SEC);
+  assert_array_equals(div.getAnimations(), [animationA, animationB]);
+}, 'Test getAnimations on element with two animations');
+
+test(function(t) {
+  var divA = createDiv(t);
+  var divB = createDiv(t);
+  var animationA = divA.animate(null, 100 * MS_PER_SEC);
+  var animationB = divB.animate(null, 100 * MS_PER_SEC);
+  assert_array_equals(divA.getAnimations(), [animationA], 'divA');
+  assert_array_equals(divB.getAnimations(), [animationB], 'divB');
+}, 'Test getAnimations on separate elements with separate animations');
+
+test(function(t) {
+  var divParent = createDiv(t);
+  var divChild = createDiv(t);
+  divParent.appendChild(divChild);
+  var animation = divChild.animate(null, 100 * MS_PER_SEC);
+  assert_array_equals(divParent.getAnimations(), [], 'divParent');
+  assert_array_equals(divChild.getAnimations(), [animation], 'divChild');
+}, 'Test getAnimations on element with animation on child element');
+
+test(function(t) {
+  var div = createDiv(t);
+  var animation = div.animate(null, 100 * MS_PER_SEC);
+  animation.finish();
+  assert_array_equals(div.getAnimations(), []);
+}, 'Test getAnimations on element with finished fill none animation');
```

Comment:
It looks like the Blink test file also tests the case where the animation hasn't started having an effect yet (by having a delay set). Add that in too?